proposals/NNNN-font-styles.md

```diff
@@ -0,0 +1,110 @@
+# Font Styles
+
+* Proposal: [SDL-NNNN](NNNN-font-styles.md)
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / Web / RPC ]
```

Comment:
@MichaelCrimando can you please update `Android` to `Java Suite` and specify the `Web` platforms impacted?  For example, Policy Server, SHAID, JavaScript Suite?